.github/workflows/build.yml

```
@@ -0,0 +1,15 @@
+name: Build images
+
+on: [workflow_dispatch]
```

```suggestion
on: workflow_dispatch
```

Unless you intend to add other actions, you can take this out of a list as demonstrated here: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch